lib/stripe/stripe_object.rb

```diff
@@ -249,7 +256,24 @@ def method_missing(name, *args)
     end
 
     def respond_to_missing?(symbol, include_private = false)
-      @values && @values.has_key?(symbol) || super
+      if symbol.to_s.end_with?('?')
+        name = chop_symbol(symbol)
+        has_boolean_value?(name) || super
+      else
+        @values && @values.has_key?(symbol) || super
+      end
     end
+
+    # Returns a new symbol with the last character removed.
+    # e.g. `:foo=` becomes `:foo`.
+    def chop_symbol(symbol)
+      symbol.to_s[0...-1].to_sym
```

Comment:
Kind of a bikeshed, but I wonder if we should just update this to `symbol.to_s.chop.to_sym` as long as we're making this change.